packages/terra-avatar/docs/upgrade-guide.md

```diff
@@ -1,4 +1,36 @@
 # terra-avatar Upgrade Guide
+## Changes from version 2 to version 3
+
+### Removed
+* `Shared User` Variant From Avatar Variants
+
+### Added
+* `Generic` Variant with props same as `Shared User` and with `Variant` prop to support sub-variants like `single-user`, `shared-user` and `provider`.
+* `variant` prop. this will take values for sub-variants `single-user`, `shared-user` and `provider`.
```

Comment:
This is confusing to me as I understand that you are probably talking about the `variant` prop of `Generic` sub-component but at first glance, I felt this is in contradiction to [what was mentioned earlier](https://github.com/cerner/terra-core/blob/master/packages/terra-avatar/docs/upgrade-guide.md#removed). 

I'd remove this line altogether.

Reply:
Replaced line with suggested line below..!